src/app/containers/ArticleMain/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import { string, object } from 'prop-types';
+import { C_OAT_LHT } from '@bbc/psammead-styles/colours';
+import MetadataContainer from '../Metadata';
+import headings from '../Headings';
+import text from '../Text';
+import image from '../Image';
+import Blocks from '../Blocks';
+import Timestamp from '../Timestamp';
+import {
+  layoutGridWrapper,
+  layoutGridItemConstrained,
+} from '../../lib/layoutGrid';
+
+const Wrapper = styled.div`
+  ${layoutGridWrapper};
+`;
+
+const OatWrapper = styled(Wrapper)`
+  background: ${C_OAT_LHT};
+`;
+
+const GridItemConstrained = styled.div`
+  ${layoutGridItemConstrained};
+`;
+
+const componentsToRenderHeadline = {
+  headline: headings,
+};
+
+const componentsToRenderMain = {
+  subheadline: headings,
+  text,
+  image,
+};
+
+const splitBlocksByHeadline = ({ model }) => {
+  const { blocks } = model;
+
+  const headlineIndexPlusOne =
+    blocks.findIndex(({ type }) => type === 'headline') + 1;
+
+  const headlineBlocks = blocks.slice(0, headlineIndexPlusOne);
+  const mainBlocks = blocks.slice(headlineIndexPlusOne, blocks.length);
+
+  return { headlineBlocks, mainBlocks };
+};
+
+const ArticleMain = ({ service, articleData }) => {
+  const { content, metadata, promo } = articleData;
+  const { headlineBlocks, mainBlocks } = splitBlocksByHeadline(content);
+
+  /*
+   * headlineBlocks length check is temporary
+   * Simorgh will respond with 400 to lack of headline block in issue
+   * https://github.com/bbc/simorgh/issues/836
+   */
+  if (headlineBlocks.length > 0) {
```

Comment:
I've kept this logic as altering it seems out of scope for this PR. However, I'm not sure we intend to enforce it any more, & even if we do, returning 4XX seems incorrect -- this isn't a user/request error.

If we wanted to enforce it, following this PR we should be able to simply throw an error & return a 500.